custom/icds_reports/ucr/data_sources/thr_forms_v2.json

```diff
@@ -121,6 +139,22 @@
           "type": "constant",
           "constant": 1
         }
+      },
+      {
+        "display_name": null,
+        "transform": {},
+        "datatype": "string",
+        "expression": {
+          "type": "property_path",
+          "property_path": [
+            "form",
+            "photo_thr_packets_distributed"
+          ]
+        },
+        "is_primary_key": false,
+        "is_nullable": true,
+        "column_id": "image_name",
```

Comment:
I think this would be more obvious if named `photo_thr_packets_distributed`

Reply:
Good point!. Changed the name.